config/config_tests.xml

```diff
@@ -244,7 +244,12 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <BFBFLAG>TRUE</BFBFLAG>
     <STOP_OPTION>ndays</STOP_OPTION>
     <STOP_N>11</STOP_N>
+    <REST_N>$STOP_N / 2 + 1</REST_N>
```

Comment:
@billsacks I can understand that, but this approach is consistent with ERS and other tests.